.github/workflows/deb-builder.yml

```diff
@@ -0,0 +1,77 @@
+on: push
+
+jobs:
+  build-debs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: leonheldattoradex/build-deb-action@v1.7.1
+        env:
+          DEB_BUILD_OPTIONS: noautodbgsym
+        with:
+          buildpackage-opts: --build=binary --no-sign
+          host-arch: arm64
+          apt-opts: --install-recommends
+          apt-sources: |
+            Types: deb
+            URIs: https://feeds.toradex.com/debian
```

Comment:
Note here that we don't use the snapshots, because the package builds don't have to be stable in regards to the feed: the snapshots only exist for the containers.

Reply:
Cool!

Reply:
True. I added here because it's necessary for the GitHub Action, otherwise it 404s feeds.toradex.com. I'll move to the deb-builder.yml script instead.